modules/custom/sayan/sayan.info.yml

```diff
@@ -0,0 +1,8 @@
+name: sayan
+description: This is a custom module for sayan.
+package: custom
+type: module
+core_version_requirement: ^9.2 || ^10
+
+dependencies:
```

Comment:
Any specific reason why paragraph is listed as a dependency here ?

Reply:
No specific dependencies were mentioned there so I added a random one for the task
.